tests/modules/test_hspsmswrapper.py

```diff
@@ -0,0 +1,8 @@
+from cshsms.settings import HSPSMS_API, HSPSMS_USERNAME, HSPSMS_SENDERNAME
+from django.test import TestCase
+from modules.hspsmswrapper import Hspsms
+
+class HspsmsSendingTests(TestCase):
+	def test_create_objects(self):
+		hspsms = Hspsms(apikey=HSPSMS_API, username=HSPSMS_USERNAME, sendername=HSPSMS_SENDERNAME)
+		self.assertIsInstance(hspsms, Hspsms)
```

Comment:
I think we should add some tests with `get_url_response` mocked so we can test the logic a bit without having to run our expensive live tests. I can help you with those mocks if you'd like.